docs/_manage/integration-changes/essential-tips-iq.md

```diff
@@ -1,10 +1,11 @@
 ---
-title: Rollout cadence and integration quality health tips
+title: Essential Tips for Integrating Quality Health Practices
```

Comment:
Developer Style guide recommends sentence case - Essential tips for integrating quality health practices